providers/implementations/ciphers/cipher_aes_cbc_hmac_sha.c

```diff
@@ -334,6 +334,14 @@ static void *aes_cbc_hmac_sha1_newctx(void *provctx, size_t kbits,
     return ctx;
 }
 
+static void *aes_cbc_hmac_sha1_dupctx(void *provctx)
+{
+
+    PROV_AES_HMAC_SHA1_CTX *ctx = provctx;
+
+    return OPENSSL_memdup(ctx, sizeof(*ctx));
```

Comment:
Elsewhere there are `if (ctx == NULL)` checks.  But not here.

Reply:
sorry, fixed